Phonebook.Source.MockPeopleBackend/index.js

```diff
@@ -29,6 +29,7 @@ let longOrgUnits3 = longOrgUnits.slice(31, 50);
 let cities = generateArray(15, faker.address.city);
 
 let addresses = generateArray(30, faker.address.streetAddress);
+let roomNumbers = ['1A', '2A', '3A', '4A', '5A', '6A', '1B', '2B', '3B', '4B', '5B', '6B'];
```

Comment:
please revert the changes in this file. They are not relevant for the kubernetes deployment